test/utilities/FixtureIO.js

```diff
@@ -0,0 +1,26 @@
+import domino from 'domino'
+import fs from 'fs'
+import path from 'path'
+
+/**
+ * Gets string from file in 'test/fixtures/'
+ * @param  {!string} fileName   Name of file in 'test/fixtures/'
+ * @return {?string}            String from `test/fixtures/${fileName}`
+ */
+const stringFromFixtureFile = (fileName) => {
+  return fs.readFileSync(path.resolve(__dirname, `../fixtures/${fileName}`), 'utf8')
+}
+
+/**
+ * Gets Domino document from file in 'test/fixtures/'
+ * @param  {!string} fileName   Name of file in 'test/fixtures/'
+ * @return {?document}          Domino document from `test/fixtures/${fileName}`
```

Comment:
I believe this should be `Document` with a capital D: https://developer.mozilla.org/en-US/docs/Web/API/Document